pkg/chartutil/requirements_test.go

```diff
@@ -426,7 +428,12 @@ func TestDependentChartWithSubChartsHelmignore(t *testing.T) {
 }
 
 func TestDependentChartsWithSubChartsSymlink(t *testing.T) {
-	c, err := Load("testdata/joonix")
+	joonix := filepath.Join("testdata", "joonix")
+	if err := os.Symlink(filepath.Join("..", "..", "frobnitz"), filepath.Join(joonix, "charts", "frobnitz")); err != nil {
```

Comment:
note for the reviewer: there appears to be a bug where symlinking a directory on Linux, committing that change and then testing [if it's a symbolic link](https://github.com/helm/helm/blob/3a8a797eab0e1d02456c7944bf41631546ee2e47/pkg/sympath/walk.go#L112-L115) on a Windows box fails. The fix for this is to symlink the directory on a Windows box for Windows testing. That seems to work to get the tests passing again, and I don't think there's a good way to fix this.